ReSharper.FSharp/test/data/features/daemon/typeHints/Settings 01 - Show all.fs.gold

```diff
@@ -6,37 +6,43 @@ let f x||(0) ||(1)=
     let g x||(2) ||(3)= ()
     let x||(4) = fun y||(5) -> ()
     for x||(6) in [""] do ()
+    match 5 with x||(7) -> () 
     { new IFormattable with
-        member x.ToString(format||(7), provider||(8)) ||(9)= "" }
+        member x.ToString(format||(8), provider||(9)) ||(10)= "" }
 
-fun x||(10) -> ()
+fun x||(11) -> ()
 
-type A(x||(11)) =
+match 5 with
+| x||(12) when let y||(13) = 5 in true ->
+    let z||(14) = 3
+    ()
+
+type A(x||(15)) =
     do
-        let x||(12) = 3
+        let x||(16) = 3
```

Comment:
@DedSec256 This seems unrelated to match clauses. Could you split these test cases, please?

Reply:
This is a special test case where all hints are enabled and contains the minimum set of each hint category. Previously I split tests into smaller ones here
https://github.com/JetBrains/resharper-fsharp/commit/313392c557b7e5c7ef2774676a40de77c9196e05